app/src/components/Drawer.vue

```diff
@@ -83,19 +83,26 @@
           </router-link>
         </md-list>
       </md-list-item>
-      <md-list-item>
+      <md-list-item v-if="isAuth" @click="$store.dispatch('auth/logout')">
         <md-icon class="utility-navfonticon">logout</md-icon>
         <span class="md-list-item-text utility-navfont">Log out</span>
       </md-list-item>
-      <md-list-item>
+      <md-list-item v-if="!isAuth">
         <md-icon class="utility-navfonticon">login</md-icon>
-        <span class="md-list-item-text utility-navfont">Login</span>
+        <span class="md-list-item-text utility-navfont"><a href="/secure">Login</a></span>
```

Comment:
I meant to actually comment on this one: 
Here's how it looks on mine
<img width="409" alt="Screen Shot 2023-02-21 at 3 22 16 PM" src="https://user-images.githubusercontent.com/24441706/220481251-e0da382d-2557-4463-99b5-dca3970644b2.png">

Reply:
Oh yes, we should. I noticed that too but planned to handle it separately. Also we can combine that ticket with the Search secondary page. The initial fix I had was for the search primary page and not the secondary. I will create the ticket and take care of it